docs/azure/sdk/protocol-convenience-methods.md

```diff
@@ -96,14 +95,15 @@ The preceding code demonstrates the following `System.ClientModel` convenience m
 
 The following code uses a `ChatClient` to call the `CompleteChat` protocol method:
 
-:::code source="snippets/protocol-convenience-methods/SCMProtocol/Program.cs" highlight="26-31":::
+:::code source="snippets/protocol-convenience-methods/SCM/Protocol/Program.cs" highlight="26-31":::
 
 The preceding code demonstrates the following `System.ClientModel` protocol method patterns:
 
-- Uses the `BinaryContent` type as a parameter to supply data for the request body.
-- Uses the `RequestContext` type to configure request options.
-- Returns data using the `ClientResult` type.
-- Calls the <xref:System.ClientModel.ClientResult.GetRawResponse%2A> method to access the response data.
+1. Create the request, using a `BinaryContent` object for the request body.
+1. Invoke the protocol method, using a `RequestOptions` object to configure request options.
+1. Handle the response by reading:
+    - The HTTP status code from the `PipelineResponse` object to determine whether success or failure.
+    - Data from the `PipelineResponse` object's content using `System.Text.Json` APIs.
```

Comment:
nit: is it worth saying "or Azure.Core APIs"?  (to the extent that `ToDynamicFromJson` can be used in place of STJ APIs...).  Not critical as it is mentioned below.

Reply:
The only reason I didn't do this is because the focus is on the approach the sample code took.

Reply:
Makes sense!